### PR TITLE
Base setup for loading pretrained models

### PR DESCRIPTION
**Pull request by PraneetNeuro**

### CODE REVIEW

File: src/vit_prisma/configs/HookedViTConfig.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ class HookedViTConfig:
     normalization_type: Optional[str] = "LN"
     device: Optional[str] = None
     n_devices: int = 1
-    attention_dir: str = "causal"
+    attention_dir: str = "bidirectional"
```
Comment (themachinefan) [inline src/vit_prisma/configs/HookedViTConfig.py:38]: Good catch

File: src/vit_prisma/models/base_vit.py
Comment (soniajoseph) [inline src/vit_prisma/models/base_vit.py:1]: We should probably make is_timm an argument, or find a way of specifying the model name (not all models are timm), as per the TransformerLens setup

Comment (soniajoseph) [inline src/vit_prisma/models/base_vit.py:1]: Right now, is_timm is hardcoded, but we should find a way to flexibly specify the model name (and not all models will be timm).

File: src/vit_prisma/utils/prisma_utils.py
Comment (soniajoseph) [inline src/vit_prisma/utils/prisma_utils.py:1]: We should also move any weight conversion functions into a separate document (not prisma_utils)

as follows
https://github.com/neelnanda-io/TransformerLens/blob/main/transformer_lens/loading_from_pretrained.py

File: src/vit_prisma/models/base_vit.py
Comment (soniajoseph) [reply src/vit_prisma/models/base_vit.py:1]: If we're using TransformerLens code directly, we should cite it at the beginning of that document or on that function.

File: src/vit_prisma/utils/FactoredMatrix.py
Comment (soniajoseph) [inline src/vit_prisma/utils/FactoredMatrix.py:1]: There is already FactoredMatrix code under vit_prisma/prisma. Lol, maybe we should rename that folder something else.

File: src/vit_prisma/utils/prisma_utils.py
Comment (PraneetNeuro) [reply src/vit_prisma/utils/prisma_utils.py:1]: Moved the code to a separate file under the now named prisma folder

**Pull request merged**